### DISCUSSION TITLE
Remix command for the manual configuration of routes

### DISCUSSION THREAD

**Discussion by alamenai** [remix-run/remix#10290] [Proposals]
Hi Remix!

I'm new to Remix and I just started learning it this weekend, I already started falling in love with it and it was easy to understand. Though I need time to understand the file convention for routing, I liked the section of [Manual Configuration](https://remix.run/docs/en/main/discussion/routes#manual-route-configuration), and I would suggest if there is a possibility we can have a Remix CLI that generates the configuration from the file structures instead of doing it manually.

Thank you,

**Comment by sergiodxa** [comment]
https://github.com/sergiodxa/remix-utils/discussions/417#discussioncomment-11416439
